test/updating-stage-throttling-settings.js

```diff
@@ -120,7 +120,7 @@ describe('Updating stage throttling settings', () => {
       });
 
       it('should specify two patch operations per endpoint with custom throttling settings and two patch operations for the stage', () => {
-        expect(apiGatewayRequest.properties.patchOperations).to.have.lengthOf(6);
+        expect(apiGatewayRequest.properties.patchOperations).to.have.lengthOf(8);
```

Comment:
The description of this test would also need to be something like:

`should specify two patch operations for each http endpoint and two patch operations for the stage`